apps/tangle-cloud/src/pages/instances/Instances/UpdateBlueprintModel/ApproveConfirmationModal.tsx

```diff
@@ -0,0 +1,263 @@
+import { MonitoringServiceRequest } from '@tangle-network/tangle-shared-ui/data/blueprints/utils/type';
+import {
+  Button,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+} from '@tangle-network/ui-components';
+import BlueprintItem from '@tangle-network/tangle-shared-ui/components/blueprints/BlueprintGallery/BlueprintItem';
+import { ApprovalConfirmationFormFields } from '../../../../types/approvalConfirmationForm';
+import { useForm } from 'react-hook-form';
+import { Children, useRef } from 'react';
+import { RestakeAssetId } from '@tangle-network/tangle-shared-ui/types';
+import { AssetCommitmentFormItem } from './AssetCommitmentFormItem';
+import { PalletAssetsAssetMetadata } from '@polkadot/types/lookup';
+
+type ApproveConfirmationModelProps = {
+  onClose: () => void;
+  onConfirm: (data: ApprovalConfirmationFormFields) => Promise<boolean>;
+  selectedRequest: MonitoringServiceRequest | null;
+  assetsMetadata: Map<string, PalletAssetsAssetMetadata | null>;
+};
+
+function ApproveConfirmationModel({
```

Comment:
Typo here, also prefer using `const ...` instead of `function ...` for consistency
```suggestion
function ApproveConfirmationModal({
```